### PR TITLE
Add a clusterloader2 kubemark presubmit.

### PR DESCRIPTION
**Pull request by mm4tt**
To avoid issues like https://github.com/kubernetes/kubernetes/issues/78401

### CODE REVIEW

File: config/jobs/kubernetes/sig-scalability/sig-scalability-presubmit-jobs.yaml [hunk 1]
```diff
@@ -312,3 +312,55 @@ presubmits:
...
+            - --cluster=
+            - --extract=local
+            - --gcp-master-size=n1-standard-2
+            - --gcp-node-image=gci
+            - --gcp-node-size=n1-standard-4
+            - --gcp-nodes=4
+            - --gcp-project=k8s-presubmit-scale
+            - --gcp-zone=us-east1-b
+            - --kubemark
+            - --kubemark-nodes=100
```
Comment (wojtek-t) [inline config/jobs/kubernetes/sig-scalability/sig-scalability-presubmit-jobs.yaml]: Do we want 100? Why not 500 to be comparable with k/k?

Comment (mm4tt) [reply config/jobs/kubernetes/sig-scalability/sig-scalability-presubmit-jobs.yaml]: Done.

**Pull request merged**